src/realm/cluster_tree.hpp

```diff
@@ -128,17 +129,46 @@ class ClusterTree {
         m_root->remove_column(col);
     }
 
+    // Create and return object
+    Obj insert(ObjKey k, const FieldValues& values);
```

Comment:
Why don't we return a cluster tree iterator?

Reply:
Because what we need is an Obj.